app/src/test/java/bitmap/transformer/AppTest.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package bitmap.transformer;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
```

Comment:
Don't forget to test out your methods. You could always test assertTrue's or null values.